src/design-tokens/tier-3-component/forms.json

```diff
@@ -5,13 +5,10 @@
         "form": {
           "border": {
             "@": {
-              "value": "{eds.color.neutral.300}"
+              "value": "{eds.color.neutral.500}"
             },
             "hover": {
-              "value": "{eds.color.neutral.400}"
-            },
-            "disabled": {
-              "value": "{eds.color.neutral.100}"
```

Comment:
In https://github.com/chanzuckerberg/edu-design-system/pull/1279 I darkened the `color-border-disabled` token for buttons and added a separate `color-form-border-disabled` token here to avoid changing the input field styles until I talked to design about it. Since the input field disabled border is now in sync with the others, I removed this token and went back to using the general `color-border-disabled` token in the input field styles.